sdk/storage/azure-storage-queue/CHANGELOG.md

```diff
@@ -1,6 +1,22 @@
 # Release History
+## Version 12.0.0-preview.4 (2019-10-8)
```

Comment:
Same feedback applies to this file
1. update Java Beans and Module support text to match the other 2 changelogs
2. quote `code` in this file to be consistent with the blobs changelog

Reply:
Done.